.ci_support/linux_mpimpich.yaml

```diff
@@ -14,12 +12,22 @@ fortran_compiler:
 - gfortran
 fortran_compiler_version:
 - '7'
+hdf5:
+- 1.10.5
+libblas:
+- 3.8 *netlib
```

Comment:
Hmm, won't this rely on netlib blas?

Reply:
If openblas or mkl are available they are used instead. I was basically just following https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#blas

Reply:
Hmm, the way I read the documentation you should just have `libblas` no version specification, nor `*netlib`? It has changed some times during the anaconda period. I have tried to adapt, and now it seems to have changed again ;)

Hmm. I can try this when I get the time.

Reply:
I use the same for quantum espresso and there it works fine https://github.com/conda-forge/qe-feedstock/blob/master/.ci_support/linux_.yaml

Reply:
Yes, it works, but is it netlib BLAS?

Reply:
No, here is the quantum espresso example: 
https://mybinder.org/v2/gh/jan-janssen/quantum-espresso-example/master
By default openblas is installed. The netlib part is only used during the compilation to make sure no openblas specific commands are used and the executable can run with mkl, openblas and netlib.

Reply:
```

libblas                   3.8.0               14_openblas    conda-forge
libcblas                  3.8.0               14_openblas    conda-forge
...
qe                        6.5                  h2fbfb5d_0    conda-forge
```

Reply:
Ok, thanks for the clarification! :)